content/apps/custom-domains.md

```diff
@@ -152,7 +94,8 @@ cloud.gov offers a managed service that allows you to forward a domain you contr
     Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
     ```
 
-1. Create/update your DNS configuration.
+1. Create/update your DNS configuration (including your IPv6 AAAA configuration if you need it).
```

Comment:
Sorry for the confusion... ipv6 is not available in cloudfront just in ELBs so we have to use the "manual method" to make it work.